site/developer/model-documentation/generating-model-documentation.qmd

```diff
@@ -0,0 +1,101 @@
+---
+title: "Generating model documentation"
```

Comment:
```suggestion
title: "Documenting models"
```

This topic covers more than just _documentation generation_ which is just step 2. in what's being described. A more encompassing title would be the suggestion I made, but you will need to update the title and any related stuff like _quarto.yml and links/listings.